tests/agent/test_consistent_trajectory.py

```
@@ -359,99 +457,38 @@ async def test_falco_agent_consistent_trajectory_with_completion():
-                    raise ValueError(f"Unknown message content type: {type(m['content'])}")
-
 
-@pytest.mark.asyncio
-async def test_falco_agent_step_callback():
-    """Test that step callback is called for each step"""
-
-    callback_calls = []
-
-    def step_callback(response: AgentStepResponse) -> None:
-        callback_calls.append(response)
+            with open(OUTPUT_FAIL_COMPLETION_MESSAGES_FILE, "w") as f:
```

Ruff (0.12.2)

Undefined name `OUTPUT_FAIL_COMPLETION_MESSAGES_FILE`

(F821)